private-templates-service/client/src/components/Dashboard/Dashboard.tsx

```diff
@@ -96,6 +96,7 @@ class Dashboard extends React.Component<Props> {
   selectTemplate = (templateID: string) => {
     this.props.history.push("preview/" + templateID);
   };
+
```

Comment:
why did u add one newline lol

Reply:
styling choice  

it looked like this b4: 
};
render(){...

Reply:
lol